website/landingpage/dev/src/assets/js/app.js

```diff
@@ -154,10 +154,42 @@ function removeActiveState(arr) {
 
 // BLOG
 // use history api back() instead of standard link if coming from overview page
-if (jQuery('.back-to-blog').length && document.referrer.indexOf('/blog/overview') !== -1 && window.history) {
-  jQuery('.back-to-blog').on('click', function(e) {
+if ($('.back-to-blog').length && document.referrer.indexOf('/blog/overview') !== -1 && window.history) {
+  $('.back-to-blog').on('click', function(e) {
     e.preventDefault();
     e.stopPropagation();
     history.back();
   });
-}
\ No newline at end of file
+}
```

Comment:
Since you touched it. How about declare `var backToBlog = $('#back-to-blog');` and move the logic part to line 171? That will be more consistent.